dev-tools/omdb/tests/test_all_output.rs

```diff
@@ -480,9 +479,9 @@ async fn do_run_extra<F>(
         cmd_path.file_name().expect("missing command").to_string_lossy(),
         args.iter()
             .map(|r| {
-                extra_redactions.map_or_else(
+                redactor.map_or_else(
                     || r.to_string(),
-                    |redactions| redact_extra(r, redactions),
+                    |redactor| redactor.do_redact(r),
                 )
```

Comment:
Something that comes to mind as I read this is that I think we can simplify usage quite a bit in this file if `Option<Redactor>` could be replaced with a `Redactor` that does nothing by default. I think `#[derive(Default)]` on `Redactor` would do the right thing?

Reply:
Yeah that's reasonable.

Reply:
Done. I think the default should be that redactions are enabled, so I added a manual `Default` impl as well as a noop constructor.